app/assets/stylesheets/darkswarm/shop.scss

```diff
@@ -67,7 +71,13 @@
     product {
       @include csstrans;
 
-      border-top: 1px solid #e5e5e5;
+      // Avoid margin collapsing which breaks the flush alignment of the first
+      // image. https://stackoverflow.com/a/19719427/3377535
```

Comment:
I dont think the SO article is specific enough to deserve to be here.
I'd keep the comment only, any dev can google "avoid margin collapse" and find that article or similar.